resolwe_bio/processes/import_data/sra_file.py

```diff
@@ -76,6 +76,12 @@ class Input:
         class Advanced:
             """Advanced options."""
 
+            prefetch = BooleanField(label="Prefetch SRA file", default=True)
+            max_size_prefetch = StringField(
+                label="Maximum file size to download in KB",
```

Comment:
Label should indicate that this is in relation to the `Prefetch` option. Ideally, we would hide this option when `prefetch = False`.